src/model/Model.cpp

```diff
@@ -179,7 +178,7 @@ void Model::generateSQLParts(string &sqlHead, string &sqlBatch, string &sqlUpdat
     }
 }
 
-string Model::generateSQLTail(string sqlBatch, string sqlUpdate, int batchsize, Mode mode) {
+string Model::generateSQLTail(string sqlBatch, string &sqlUpdate, int batchsize, Mode mode) {
```

Comment:
Hmm, not sure how I missed that. 
Have updated this so that `sqlBatch `and `sqlUpdate `are both passed by reference